src/modules/Login/action.js

```diff
@@ -180,6 +183,30 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
     accountInitObject.pinMode = coreFinal.pinMode
     accountInitObject.otpMode = coreFinal.otpMode
 
+    // Check clock skew against atomic time
+    try {
+      const remoteTime = await fetch('http://worldtimeapi.org/api/timezone/Etc/GMT')
+      const body = await remoteTime.json()
```

Comment:
Do not put a blocking network call in initAccount as this can slow down login times for users. This should run in the background and throw a modal once complete.

Reply:
Also, we should probably have this at app startup time instead of initAccount as the user might not be able to login at all. This would entail moving almost the entire PR to the edge-login-ui repo which is a good idea so SDK partners get it as well.